build.psm1

```diff
@@ -538,6 +538,7 @@ Fix steps:
     $config = @{}
     if ($environment.IsWindows) {
         $config = @{ "Microsoft.PowerShell:ExecutionPolicy" = "RemoteSigned" }
+        $config += @{ "WindowsPowerShellCompatibilityModuleDenyList" = @("PSScheduledJob") }
```

Comment:
This is just a hashtable, you should just extend line 540 rather than doing an append operation

Reply:
Fixed. Also added 'BestPractices' module to the list because as mentioned in #11202 it doesn't currently work and breaks other modules in the WinCompat session.